src/lib/apigee/developerApps.js

```diff
@@ -18,6 +18,20 @@ class DeveloperApps {
     }
   }
 
+  async getCustomAttribute(developer, app, attributeName) {
+    console.log(`/organizations/${this.config.organization}/developers/${developer}/apps/${app}/attributes/${attributeName}`);
+    try {
+      const customAttributes = await this.request.get(`/organizations/${this.config.organization}/developers/${developer}/apps/${app}/attributes/${attributeName}`)
+      console.log(customAttributes.data.value)
```

Comment:
return the value instead of console logging it here, this method should only GET the attribute. The logic for outputting it to the CLI should be added to the src/developerApps.js